src/App.tsx

```diff
@@ -17,18 +49,38 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                onSelect={handleSelectFilter}
+                onInput={handleInputFilter}
+                query={query}
+                onCross={handleClearInputBtn}
```

Comment:
```suggestion
                onClickClearButton={handleClickClearButton}
```
`onCross` doesn't explainable enough